AggregateMetrics/AggregateMetrics/AzureWebApp/Implementation/SdkVersionAzureWebApp.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.AzureWebApp
+{
+    internal class SdkVersionAzureWebApp
+    {
+        internal const string VersionPrefix = "azwpac: ";
```

Comment:
My reading of this was AZure Wep Apps Performance Counters. So azwapc, not azwpac